spec/proceduralSpec.php

```diff
@@ -0,0 +1,13 @@
+<?php
+namespace procedural\spec;
+
+use function prodecural\echoScript;
+
+describe('procedural', function () {
+  describe('->echoScript()', function () {
+    it('echos input string inside a script tag', function () {
```

Comment:
please check .travis.yml https://github.com/sitrunlab/learn-kahlan/blob/master/.travis.yml . I'm personally ok with removing php 5.5 ( to be `^5.6|^7.0` ),  but consider that kahlan still support php 5.4.